server/controllers/request.js

```diff
@@ -0,0 +1,53 @@
+require('dotenv').config();
+const Request = require('../models/Request');
+const asyncHandler = require('express-async-handler');
+
+// @route GET /request
+// @desc get requests related to a logged in user or sitter
+// @access Private
+exports.getRequest = asyncHandler(async (req, res) => {
+  const userId = req.user.id;
+  const getUser = await Request.find({userId: req.user.id});
+  if(getUser){
+    res.send(200).json({ success:{ message:  "get the " + getUser }});
+  } else {
+    res.status(404).send("user didn't exist.");
+  }
+});
+
+// @route POST /request
+// @desc post requests with user id and sitter id
+// @access Private
+exports.postRequest = asyncHandler(async (req, res) => {
+  const { sitterId, start, end, offset } = req.body;
+  const userId = req.user.id;
+  const getUser = await Request.find({userId: userId});
+
+  if(getUser){
+    res.status(409).send("user is exist, can not be duplicate");
```

Comment:
Please add an exception here or put a return statement, becuase if there any code below it or after and not if conditional block it will executed

Reply:
I don't understand why we are preventing a user from creating multiple booking requests? This shouldn't be here and a user can create as many requests as they want for a pet sitter.

Reply:
I see so user can have more than two sitter id